src/poliastro/integrators.py

```diff
@@ -0,0 +1,185 @@
+from scipy.integrate import OdeSolver, DenseOutput
```

Comment:
This whole module is just impressive. Good job!

Reply:
thanks) that was a great thing to me to recall how all these finite-difference suffisticated methods work